pkg/disttask/framework/testutil/context.go

```diff
@@ -166,13 +167,14 @@ func (c *TestDXFContext) ScaleOut(nodeNum int) {
 // ScaleOutBy scales out a tidb node by id, and set it as owner if required.
 func (c *TestDXFContext) ScaleOutBy(id string, owner bool) {
 	c.T.Logf("scale out node of id = %s, owner = %t", id, owner)
-	exeMgr, err := taskexecutor.NewManager(c.Ctx, id, c.TaskMgr)
+	nodeRes := proto.NewNodeResource(16, 32*units.GiB)
+	exeMgr, err := taskexecutor.NewManager(c.Ctx, id, c.TaskMgr, nodeRes)
```

Comment:
in `TestDXFContext`, the cpu count is a input param that's passed to `github.com/pingcap/tidb/pkg/util/cpu/mockNumCpu",`, the new node need also conform to this

see newTestDXFContext